### PR TITLE
Fix: toast not being displayed after successfully image download.

### PR DESCRIPTION
**Pull request by vishwesh3**

### CODE REVIEW

File: src/api/downloadFile.android.js [hunk 1]
```diff
@@ -5,17 +5,21 @@ import type { Auth } from '../types';
...
-    Authorization: getAuthHeader(auth.email, auth.apiKey),
-  });
+  })
+    .fetch('GET', url, {
+      'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
+      'User-Agent': userAgent,
+      Authorization: getAuthHeader(auth.email, auth.apiKey),
+    })
+    .then(onSuccess)
+    .catch(onError);
```
Comment (borisyankov) [inline src/api/downloadFile.android.js]: This is a good example where the `async`, `await` and `catch` paradigm works way better.
Just don't handle anything here and don't pass callbacks, leave the function as-is.

File: src/lightbox/LightboxActionSheet.js [hunk 2]
```diff
@@ -7,31 +7,31 @@ import Toast from '../utils/showToast';
...
 
 type ButtonType = {
   title: string,
-  onPress: (props: ButtonProps) => void | boolean,
+  onPress: (props: ButtonProps) => void | boolean
 };
 
 const downloadImage = ({ url, auth }: DownloadImageType) => {
-  download(url, auth).then(() => Toast('Download complete.'));
+  download(url, auth, () => Toast('Download complete.'), () => Toast("Can't download"));
```
Comment (borisyankov) [inline src/lightbox/LightboxActionSheet.js]: ... now you can do an `await` inside a `try { } catch {}` block and handle the error, without ever passing callbackfunctions.

File: src/lightbox/LightboxActionSheet.js [hunk 3]
```diff
@@ -7,31 +7,31 @@ import Toast from '../utils/showToast';
...
 type ButtonProps = {
   auth?: Auth,
-  url: string,
+  url: string
 };
 
 type ButtonType = {
   title: string,
-  onPress: (props: ButtonProps) => void | boolean,
+  onPress: (props: ButtonProps) => void | boolean
```
Comment (borisyankov) [inline src/lightbox/LightboxActionSheet.js]: I think your `prettier` is configured inconsistently with the rest.

File: src/api/downloadFile.android.js (hunk 1 above)
Comment (vishwesh3) [reply src/api/downloadFile.android.js]: Used `async`,` await` and `catch` 👍

File: src/lightbox/LightboxActionSheet.js (hunk 3 above)
Comment (vishwesh3) [reply src/lightbox/LightboxActionSheet.js]: Not able to get which attribute is configured wrongly 🤔

File: src/api/downloadFile.android.js [hunk 4]
```diff
@@ -8,14 +8,17 @@ import userAgent from '../utils/userAgent';
...
-    'User-Agent': userAgent,
-    Authorization: getAuthHeader(auth.email, auth.apiKey),
-  });
+  })
+    .fetch('GET', url, {
+      'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
+      'User-Agent': userAgent,
+      Authorization: getAuthHeader(auth.email, auth.apiKey),
+    })
+    .catch(new Error("Can't download"));
```
Comment (borisyankov) [inline src/api/downloadFile.android.js]: This `catch` is now redundant.

File: src/lightbox/LightboxActionSheet.js [hunk 5]
```diff
@@ -30,8 +30,13 @@ type ButtonType = {
...
 };
 
-const downloadImage = ({ url, auth }: DownloadImageType) => {
-  download(url, auth).then(() => Toast('Download complete.'));
+const downloadImage = async ({ url, auth }: DownloadImageType) => {
+  try {
+    await download(url, auth);
+    Toast('Download complete.');
+  } catch (error) {
+    Toast("Can't download");
```
Comment (borisyankov) [inline src/lightbox/LightboxActionSheet.js]: `"` => `'`

Comment (vishwesh3) [reply src/lightbox/LightboxActionSheet.js]: Ah! sorry missed it. Updated 👍

File: src/api/downloadFile.android.js (hunk 4 above)
Comment (vishwesh3) [reply src/api/downloadFile.android.js]: 👍

**Comment by borisyankov** [comment]
Looks good!

**Pull request merged**